index.html

```diff
@@ -515,13 +515,15 @@ <h2>
           Promise&lt;boolean&gt; canMakePayment();
 
           readonly attribute DOMString id;
+          readonly attribute object? methodDetails;
```

Comment:
A good test to write will be that `pr.methodDetails === pr.methodDetails`. The spec makes this requirement clear but it's something implementers sometimes miss.

Reply:
Will need to test this with Basic Card.

Reply:
Added the test you suggested:  
https://github.com/w3c/web-platform-
tests/pull/10912/commits/29a384b326fc514da854a9041cdfbde13e0eee54